frontend/src/metabase/search/components/SearchUserPicker/SearchUserPicker.tsx

```diff
@@ -52,17 +56,42 @@ export const SearchUserPicker = ({
   return (
     <SearchFilterPopoverWrapper
       isLoading={isLoading}
-      onApply={() => onChange(selectedUserId)}
+      onApply={() => onChange(selectedUserIds)}
     >
       <SearchUserPickerContainer p="sm" h="100%" spacing="xs">
-        <TextInput
-          size="md"
-          mb="sm"
-          placeholder={t`Search for users…`}
-          value={userFilter}
-          tabIndex={0}
-          onChange={event => setUserFilter(event.currentTarget.value)}
-        />
+        <SearchUserSelectBox spacing="xs" p="xs">
+          {selectedUserIds.map(userId => {
+            const user = users.find(user => user.id === userId);
+            return (
+              <SelectedUserButton
+                key={userId}
+                c="brand.1"
+                px="md"
+                py="sm"
+                maw="min(30ch, 100%)"
+                rightIcon={<Icon name="close" />}
+                onClick={() =>
+                  setSelectedUserIds(without(selectedUserIds, userId))
+                }
+              >
+                <Text align="left" w="100%" truncate c="inherit">
+                  {user?.common_name}
+                </Text>
+              </SelectedUserButton>
+            );
+          })}
+
+          <SearchUserPickerInput
+            variant="unstyled"
+            size="md"
+            placeholder={
+              selectedUserIds.length === 0 ? t`Search for users…` : undefined
```

Comment:
I have mixed feelings on whether we should have a placeholder when there's more than 1 user. Have a go and let me know how it feels